server/routes/pool-management/pool-overview/pool-overview.controller.js

```diff
@@ -618,7 +618,10 @@ function courtView(app, req, res, pool, membersList, _errors, selectedJurors, se
 module.exports.postBulkPostpone = function(app) {
   return async function(req, res) {
     if (req.body['check-all-jurors']) {
-      req.body.selectedJurors = await poolMembersDAO.get(req, req.params.poolNumber);
+      const poolMembers = await poolMembersDAO.get(req, req.params.poolNumber);
```

Comment:
we should probably wrap this in a try catch to handle any error the api could throw at us

Reply:
@ricardofreitasrocha, thanks I missed that. I've now added the try catch block.